### ISSUE TITLE
Fix Analytics across GLCWRA viewers

### ISSUE THREAD

**Issue by HansVraga** [USGS-WiM/ORA#21]
Looks like some of the GA codes aren't matched correctly. Review codes and also that we've properly hooked in analytics. No WMA analytics on this, but we should also do USA analytics. 

Screenshot of the different GA codes - please review! 

![image](https://user-images.githubusercontent.com/12058013/135335738-98f45ed9-a174-4ca4-bdc9-47c38adcf6b6.png)

**Comment by lprivette** [comment]
You mean the codes in the application are not lining up with there values in the screenshot above?

**Comment by HansVraga** [comment]
yes. and I added you with @Kathy so you could give her a primer on how it all works. Also we thought we saw some irregularities with the code snippet you add to the client to link to GA, so can you show where to review those?

**Comment by lprivette** [comment]
ga has them to copy and paste in the property settings. curious to see the irregularities. If there are new and made with G4 they would look very different, and they would not be compatible with USA analytics

![image](https://user-images.githubusercontent.com/23640741/135359976-8c246320-eb54-4937-a492-abde30060888.png)

**Comment by lprivette** [comment]
Verified:
- [ ] CRSRA
- [ ] LORA
- [ ] WLERA
- [ ] SBRA
- [ ] GBRA

Should be good now. For the ones that I updated from ga to gtag. I'm pretty sure they were still getting data on the standard stuff like site visits, but event might not have been getting sent.
